tests/publiccloud/boottime.pm

```diff
@@ -241,10 +290,18 @@ sub measure_timings {
             first => {}, soft => {}, hard => {}
         },
     };
-    my $provider = $self->provider_factory();
 
-    # Provision the instance
-    my $instance = $provider->create_instance(check_connectivity => 0);
+    if (get_var('PUBLIC_CLOUD_QAM')) {
+        $instance         = $args->{my_instance};
+        $provider         = $args->{my_provider};
+        $self->{provider} = $args->{my_provider};    # required for cleanup
+        select_host_console();
+    } else {
+        $self->select_serial_terminal;
+        $provider = $self->provider_factory();
+        $instance = $self->{my_instance} = $provider->create_instance(check_connectivity => 0);
+    }
```

Comment:
This piece of code is now on at least 3 places exactly the same. Can you just create a `poo#` so we don't remember to refactor this please?

Reply:
https://progress.opensuse.org/issues/91016

Reply:
for this special test it's actually a bit misleading to use the interactive ssh session approach.
What we tried to cover here is the duration till we can access a PC instance after hit the `create` button.
So the thing how long it takes to to `cloud-init` and co.